tests/system/python/conftest.py

```diff
@@ -668,6 +670,29 @@ def _disable_sch(fledge_url, sch_name):
     return _disable_sch
 
 
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
You typically use autouse fixtures when you want to use them for setup/teardown only.

Reply:
Yes, I have used an autouse fixture with a function-limited scope. This ensures that it is executed at the end of every test function/case in any system test, allowing it to collect a support bundle before executing next test function/case.